docs/interpreter/hbase.md

```diff
@@ -6,15 +6,13 @@ group: manual
 ---
 {% include JB/setup %}
 
-
 ## HBase Shell Interpreter for Apache Zeppelin
-[HBase Shell](http://hbase.apache.org/book.html#shell) is a JRuby IRB client for Apache HBase. 
-This interpreter provides all capabilities of Apache HBase shell within Apache Zeppelin. The 
-interpreter assumes that Apache HBase client software has been installed and its possible to 
-connect to the Apache HBase cluster from the machine on where Apache Zeppelin is installed.
+[HBase Shell](http://hbase.apache.org/book.html#shell) is a JRuby IRB client for Apache HBase. This interpreter provides all capabilities of Apache HBase shell within Apache Zeppelin. The interpreter assumes that Apache HBase client software has been installed and its possible to connect to the Apache HBase cluster from the machine on where Apache Zeppelin is installed.  
+To start with HBase, please see [HBase Quickstart](https://hbase.apache.org/book.html#quickstart)
+
+> Note: currently only HBase 1.0.x releases are supported.
```

Comment:
Hi,
I have ideas on how to make it work with other versions of HBase. I'll open a JIRA and I can work supporting more versions of HBase.